src/inputs/select-input.jsx

```diff
@@ -133,46 +147,62 @@ class SelectInput extends React.Component {
 
       if (key === 40) {
         // arrow down, open and go to next opt
+        const newDownFocus =
```

Comment:
This may never occur, but what would happen if `nextValue` and `options[0]` were both falsy?

Reply:
It doesn't focus on anything. It *can* happen if you've filtered out all matching results and pressed an arrow key. In that case, we don't want the focus to change.